JustBrewIt/Pages/Brewery.cshtml.cs

```diff
@@ -19,7 +19,7 @@ public void OnGet()
             using (var webClient = new WebClient())
             {
                 string jsonString = webClient.DownloadString("https://api.openbrewerydb.org/breweries?by_city=cincinnati&brewery_type=regional");
-                Welcome[] welcome = Welcome.FromJson(jsonString);
+                BreweryAPI[] welcome = BreweryAPI.FromJson(jsonString);
```

Comment:
Change the variable name from Welcome as well.

Reply:
Changed the variable name.